engine/runtime/src/main/java/org/enso/interpreter/runtime/data/atom/GetFieldNode.java

```diff
@@ -50,7 +50,7 @@ public String getQualifiedName() {
 
   @Override
   public String getName() {
-    return type.getName() + "." + name;
+    return name;
```

Comment:
This is the fix. I can now see the fields of an atom in the debugger:
![image](https://github.com/user-attachments/assets/78fe2890-0d0d-44b5-99a2-c9f2ced57aec)